pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -490,10 +494,14 @@ protected void setupCustomRouting(@NonNull Config cfg) {
             throw new RuntimeException("appRoute was not configured correctly.");
         }
 
-        String jwtSecret = cfg.getString(ApplicationConfigConstants.BROWSER_JWT_SECRET);
         String cookieSalt = cfg.getString(ApplicationConfigConstants.BROWSER_COOKIE_SALT);
         String cookieName = cfg.getString(ApplicationConfigConstants.BROWSER_COOKIE_NAME);
```

Comment:
I think they can get removed now?

Reply:
I wasn't sure if we will need them back , I will remove